### PR TITLE
Fix UUID and integrity

### PR DESCRIPTION
**Pull request by iakovdudinsky**
Changed UUID5 (input time generating id) to UUID4 (random), fixed integrity checker errors

### CODE REVIEW

File: deker/integrity.py [hunk 1]
```diff
@@ -170,38 +170,43 @@ def check_arrays_locks(self, collection: Collection) -> None:
         if self.stop_on_error and self.errors:
             raise DekerIntegrityError(self._parse_errors())
 
-    def check(self, collection: Collection) -> None:
-        """Check if Arrays or VArray in Collection are valid.
+    def _check_v_arrays(self, collection: Collection, v_arrays: Union[ArrayManager, Optional[VArrayManager]]) -> None:
+        """Check if Arrays or VArray in Collection are initializing.
```
Comment (SerGeRybakov) [inline deker/integrity.py:174]: Just one VArray may be initialized?

File: deker/integrity.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ class PathsChecker(BaseChecker):
     CHECKER_LEVEL = 3
 
     def _validate_path_array_without_key_attributes(
-        self, main_path: Path, symlink_path: Path, files: list
+            self, main_path: Path, symlink_path: Path, files: list
```
Comment (SerGeRybakov) [inline deker/integrity.py:67]: Why this indent appeared? Have you run black before push?

File: deker/integrity.py [hunk 3]
```diff
@@ -170,38 +170,43 @@ def check_arrays_locks(self, collection: Collection) -> None:
         if self.stop_on_error and self.errors:
             raise DekerIntegrityError(self._parse_errors())
 
-    def check(self, collection: Collection) -> None:
-        """Check if Arrays or VArray in Collection are valid.
+    def _check_v_arrays(self, collection: Collection, v_arrays: Union[ArrayManager, Optional[VArrayManager]]) -> None:
+        """Check if Arrays or VArray in Collection are initializing.
 
         :param collection: Collection to be checked
+        :param v_arrays: DataManager to get arrays or varrays from collection
```
Comment (SerGeRybakov) [inline deker/integrity.py:177]: I think it's better to rename it clearly to `data_manager` if it may iterate over any collection.

**Review by SerGeRybakov — CHANGES_REQUESTED** [review]

File: deker/integrity.py [hunk 4]
```diff
@@ -170,38 +170,43 @@ def check_arrays_locks(self, collection: Collection) -> None:
         if self.stop_on_error and self.errors:
             raise DekerIntegrityError(self._parse_errors())
 
-    def check(self, collection: Collection) -> None:
-        """Check if Arrays or VArray in Collection are valid.
+    def _check_v_arrays(self, collection: Collection, v_arrays: Union[ArrayManager, Optional[VArrayManager]]) -> None:
```
Comment (matveyvarg) [inline deker/integrity.py:173]: If ArrayManger is also can be passed, it's better to rename argument as you have to deal with both types of managers

File: deker/integrity.py [hunk 5]
```diff
@@ -170,38 +170,43 @@ def check_arrays_locks(self, collection: Collection) -> None:
...
+                        self.errors[f"Collection {collection.name} arrays integrity errors:"].append(
+                            str(e)
+                        )
                     else:
                         raise DekerIntegrityError(str(e))
+        except DekerMetaDataError as e:
+            if not self.stop_on_error:
+                self.errors[f"Collection {collection.name} (V)Arrays initialization errors:"].append(str(e))
+            else:
+                raise e
```
Comment (gurich92) [inline deker/integrity.py:195]:
```
            if self.stop_on_error:
                raise e
            self.errors[f"Collection {collection.name} (V)Arrays initialization errors:"].append(str(e))
```
Might be better?

File: deker/integrity.py (hunk 3 above)
Comment (matveyvarg) [reply deker/integrity.py:177]: +1

File: deker/integrity.py [hunk 6]
```diff
@@ -170,38 +171,45 @@ def check_arrays_locks(self, collection: Collection) -> None:
         if self.stop_on_error and self.errors:
             raise DekerIntegrityError(self._parse_errors())
 
+    def _check_v_arrays(
```
Comment (matveyvarg) [inline deker/integrity.py:174]: May be rename the method name as well ?

**Review by matveyvarg — APPROVED** [review]

**Review by SerGeRybakov — APPROVED** [review]

**Pull request merged**